### PR TITLE
[cyb-4] License information added

### PR DESCRIPTION
**Pull request by kpovetkin**
License information added to sources.

### CODE REVIEW

File: stream-client/stream-client-python/v2/config.py [hunk 1]
```diff
@@ -1,3 +1,18 @@
+#  Copyright 2014 Cask Data, Inc.
```
Comment (awholegunch) [inline stream-client/stream-client-python/v2/config.py]: Copyright © 2014 Cask Data, Inc. [common]

File: pom.xml [hunk 2]
```diff
@@ -240,6 +240,11 @@
                   <exclude>stream-client-ruby/spec/fixtures/1MB</exclude>
                   <exclude>stream-client-ruby/Gemfile.lock</exclude>
                   <exclude>stream-client-ruby/README.md</exclude>
+                  <exclude>stream-client-python/v2/test/some.log</exclude>
```
Comment (awholegunch) [inline pom.xml]: You can use wildcards here to make it more robust, if that's your intent:
<exclude>stream-client-python/v2/test/*.log</exclude>

**Comment by awholegunch** [comment]
A couple of comments, once addressed will be LGTM.

**Comment by kpovetkin** [comment]
Done

**Comment by awholegunch** [comment]
LGTM

**Pull request merged**